docs/articles/azure/azs-how-resize-unmanaged-disk.md

```diff
@@ -0,0 +1,94 @@
+---
+title: How to resize an unmanaged disk | UKCloud Ltd
+description: Provides help for resizing an unmanaged disk on UKCloud for Microsoft Azure
+services: azure-stack
+author: Bailey Lawson
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to resize an unmanaged disk
+toc_fullpath: Users/How To/azs-how-resize-unmanaged-disk.md
+toc_mdlink: azs-how-resize-unmanaged-disk.md
+---
+
+# How to resize an unmanaged disk
+
+## Overview
+
+When deploying a new virtual machine from an Azure Marketplace image, the default drive size is often 127GB or less. Whilst it is recommended to add additional disks for tasks such as installing applications and for CPU intensive workloads, you may need to expand the initial drive for purposes such as migrating from a physical PC to the VM or to support applications which must be installed on the OS drive.
+
+This guide shows you how to resize disks on UKCloud for Microsoft Azure.
+
+>[!Note]

```

Comment:
Not sure if it is essential, but you may need to include a space after each greater than sign. And NOTE should be all caps

Reply:
Fixed